experiments/aws/parallel-cluster/cpu/experiment/README.md

```diff
@@ -133,13 +133,15 @@ rm singularity-ce*.tar.gz
 
 Batch script:
 
-```console
+```bash
 oras login ghcr.io --username vsoch
 cd configs/single-node-benchmarks
-srun --time=00:04 -N 2 slurm-single-node-benchmarks.sh
+# Set this to the size cluster you're running
+size=32
+srun --time=00:04 -N 64 slurm-single-node-benchmarks.sh
```

Comment:
This is srun for 64, but the job size is 32. Does this N mean something else aside from nodes?

Reply:
Fixed!